### DISCUSSION TITLE
Why is comboPatch a separate branch?

### DISCUSSION THREAD

**Discussion by isakube** [KadeArchive/Kade-Engine#884] [Q&A] [answered]
why

**Comment by Lucky-56** [comment, marked as answer]
because its a branch without any Kade Engine stuff put in, just the comboPatch (maybe a pr, was too lazy to look for that)

**Reply by Lucky-56** [reply]
I doubt that vanilla FNF would need that though (unless modding is considered)

**Reply by isakube** [reply]
oh ok

here's your daily Discussion Answered
